### PR TITLE
Update package READMEs

### PR DESCRIPTION
**Pull request by yashovardhan**
Update READMEs of all the packages and the main repository

### CODE REVIEW

**Review by shahbaz17 — APPROVED** [review]
LGTM 🐑.

READMEs look beautiful @yashovardhan 😍

File: packages/adapters/openlogin-adapter/README.md [hunk 1]
```diff
@@ -1,6 +1,118 @@
...
+# Web3Auth Openlogin Adapter
 
 [![npm version](https://img.shields.io/npm/v/@web3auth/openlogin-adapter?label=%22%22)](https://www.npmjs.com/package/@web3auth/openlogin-adapter/v/latest)
 [![minzip](https://img.shields.io/bundlephobia/minzip/@web3auth/openlogin-adapter?label=%22%22)](https://bundlephobia.com/result?p=@web3auth/openlogin-adapter@latest)
 
-Adds social logins with MFA functionality
+> Web3Auth is where passwordless auth meets non-custodial key infrastructure for Web3 apps and wallets. By aggregating OAuth (Google, Twitter, Discord) logins, different wallets and innovative Multi Party Computation (MPC) - Web3Auth provides a seamless login experience to every user on your application.
+
+
+This adapter is a wrapper around the [`openlogin`](https://github.com/torusresearch/openlogin) library from Web3Auth (previously Torus) and enables the main social login features of Web3Auth. By default, Web3Auth has
```
Comment (chaitanyapotti) [inline packages/adapters/openlogin-adapter/README.md:9]: put the link to OpenLoginSdk

File: packages/plugins/torus-wallet-connector-plugin/README.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+# Web3Auth Torus Wallet Connector Plugin
 
 [![npm version](https://img.shields.io/npm/v/@web3auth/torus-wallet-connector-plugin?label=%22%22)](https://www.npmjs.com/package/@web3auth/torus-wallet-connector-plugin/v/latest)
 [![minzip](https://img.shields.io/bundlephobia/minzip/@web3auth/torus-wallet-connector-plugin?label=%22%22)](https://bundlephobia.com/result?p=@web3auth/torus-wallet-connector-plugin@latest)
 
-Allows to inject your web3auth scoped private key into torus wallet UI (https://app.tor.us)
+> Web3Auth is where passwordless auth meets non-custodial key infrastructure for Web3 apps and wallets. By aggregating OAuth (Google, Twitter, Discord) logins, different wallets and innovative Multi Party Computation (MPC) - Web3Auth provides a seamless login experience to every user on your application.
+
+Web3Auth Torus Wallet Connector Plugin helps you get the functionalities of the Torus Solana Wallet into your Web3Auth Instance. It helps you use your Web3Auth key in the Torus Solana Wallet, making it easier for your end user to make transactions, add money into their wallet and even use it across different applications using Torus Wallet.
```
Comment (chaitanyapotti) [inline packages/plugins/torus-wallet-connector-plugin/README.md:8]: not torus solana wallet

Comment (yashovardhan) [reply packages/plugins/torus-wallet-connector-plugin/README.md]: fixed

File: packages/adapters/openlogin-adapter/README.md (hunk 1 above)
Comment (yashovardhan) [reply packages/adapters/openlogin-adapter/README.md]: changed the link to the npm package of openlogin rather.

**Comment by yashovardhan** [comment]
@chaitanyapotti I'm done with the changes.

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**